core/app/[locale]/(default)/account/(tabs)/addresses/page.tsx

```diff
@@ -24,13 +26,17 @@ export async function generateMetadata() {
   };
 }
 
+export enum AddressPageLimit {
+  Default = 3,
+}
+
 export default async function Addresses({ searchParams }: Props) {
   const { before, after } = await searchParams;
 
   const data = await getCustomerAddresses({
     ...(after && { after }),
     ...(before && { before }),
-    limit: 10,
+    limit: AddressPageLimit.Default,
```

Comment:
Also we have a default value for `limit` argument in `getCustomerAddresses` so maybe we can little bit simplify here and instead of `enum` use its default value, wdyt?💅

Reply:
Makes sense. Done!